web/src/infra/common/components/game/GameCard.tsx

```diff
@@ -1,102 +1,59 @@
+  const translate = withGameNamespace(game.code);
+  const gameName = translate('name', game.name);
+  const isFullyTranslated = makeTranslationStatusComparator(i18n.language);
 
-    let navigateButton = null;
-    const image = game.imageURL;
-    const mainDivStyle: React.CSSProperties = {
-      position: 'relative',
-      height: '250px',
-      width: '100%',
-      backgroundPosition: 'left center',
-      backgroundImage: `url(${image})`,
-      backgroundSize: 'cover',
-    };
-    const baseBadgeStyle: React.CSSProperties = {
-      position: 'absolute',
-      boxShadow: '0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24)',
-      padding: '0px 8px',
-      borderRadius: '8px',
-      backgroundColor: 'white',
-      color: 'black',
-      textDecoration: 'none',
-    };
-    if (this.props.isLink) {
-      mainDivStyle.boxShadow = '0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23)';
-      mainDivStyle.borderRadius = '8px';
-      navigateButton = (
-        <div
-          style={{
-            ...baseBadgeStyle,
-            bottom: '12px',
-            right: '8px',
-            borderRadius: '32px',
-            padding: '0',
-          }}
-        >
-          <IconButton aria-label="Next">
-            <NavigateNextIcon />
-          </IconButton>
-        </div>
-      );
-    }
-    const gameNameHeading = this.props.isLink ? (
-      <Typography gutterBottom={false} variant="h4" component="h2" style={{ fontWeight: 300 }}>
-        {t('play', { name: translate('name', game.name) })}
-      </Typography>
-    ) : (
-      <Typography gutterBottom={false} variant="h4" component="h1" style={{ fontWeight: 300 }}>
-        {translate('name', game.name)}
-      </Typography>
-    );
-    return (
-      <div style={mainDivStyle} data-testid={`gamecard-${game.code}`}>
-        <div
-          style={{
-            ...baseBadgeStyle,
-            top: '12px',
-            left: '8px',
-            paddingTop: '4px',
-            maxWidth: '500px',
-          }}
-        >
-          {gameNameHeading}
-        </div>
-        <div
-          style={{
-            ...baseBadgeStyle,
-            bottom: '12px',
-            left: '8px',
-          }}
-        >
-          <Typography gutterBottom={false} variant="overline" component="h5">
-            {translate('description', game.description)}
-          </Typography>
-        </div>
-        {navigateButton}
-      </div>
-    );
-  }
+  return (
+    <div
+      className={`${styles['Main']} ${isLink && styles['Main--link']}`}
+      style={{ backgroundImage: `url(${game.imageURL})` }}
+      data-testid={`gamecard-${game.code}`}
+    >
+      <Heading>
+        <Title>{isLink ? t('play', { name: gameName }) : gameName}</Title>
+
+        {!isFullyTranslated(game) && (
+          <Tooltip title={t('missing_translation_warning')} placement="top">
+            <a href="/docs" aria-label="Translation docs" target="_blank" onClick={(e) => e.stopPropagation()}>
```

Comment:
This is actually bringing users to `/pt/docs` instead of just `/docs`, giving a 404.

Reply:
Could you check it again? Because I've tried here and it worked.

https://user-images.githubusercontent.com/2018161/117982064-ffa5cc00-b30b-11eb-9345-a10d59a8f46d.mp4

Reply:
It works on the home page, but does not work on the GameInfo page.

We can still merge and follow up the fix in a separate PR, as it does not affect english